org.zenframework.z8.server/src/main/java/org/zenframework/z8/server/base/table/system/Tables.java

```diff
@@ -41,6 +50,9 @@ public CLASS(IObject container) {
 			setJavaClass(Tables.class);
 			setName(TableName);
 			setDisplayName(displayNames.Title);
+			Optional.ofNullable(apiAttrs.Title)
+					.ifPresent(attrVal -> setAttribute("APIDescription", attrVal));

```

Comment:
Write without lambda (->), declare APIDescription constant in org.zenframework.z8.server.json.Json